### ISSUE TITLE
when using  binary:logloss as the loss function,what's the leaf score means?

### ISSUE THREAD

**Issue by Simon-Liu622** [dmlc/xgboost#1035]
first i thought the score on the leaves are prob,but they are not?so what's the scores mean?

**Comment by pommedeterresautee** [comment]
It s the purity of the groups after the split.
Please check http://xgboost.readthedocs.io/en/latest/model.html
